src/node_internals.h

```diff
@@ -32,6 +32,7 @@
 #include "uv.h"
 #include "v8.h"
 #include "tracing/trace_event.h"
+#include "tracing/traced_value.h"
```

Comment:
Can we include these headers where they are used, rather than (almost) globally?

Reply:
Sure